pegasus/migrations/80_add_purpose_and_region_columns_to_hoc_survey_prizes.rb

```diff
@@ -0,0 +1,18 @@
+Sequel.migration do
+  up do
+    alter_table(:hoc_survey_prizes) do
+      # can't be null for unique constraint to work. Use '' for unclaimed and legacy-claimed prizes
+      add_column :purpose, String, size: 255, null: false, default: ''
+      add_index [:claimant, :purpose], unique: true
+      drop_index :claimant
+    end
+  end
+
+  down do
+    alter_table(:hoc_survey_prizes) do
+      drop_column :purpose
```

Comment:
Will the drop_column actually succeed? I think you might have to drop the index first